faqs/galaxy/galaxy_creating_an_account.md

```diff
@@ -0,0 +1,17 @@
+---
+title: How to create an account on a public Galaxy instance?
+area: Account
+layout: faq
+box_type: tip
+contributors: [jennaj, bernandez]
+---
+
+
+1. To create an account at any public Galaxy instance, choose your server from the available [list of Galaxy Platform](https://galaxyproject.org/use/).
+ There are 3 public  Galaxy servers, which are [UseGalaxy.eu](https://usegalaxy.eu/), [UseGalaxy.org (Main)](https://usegalaxy.org/) and [UseGalaxy.org.au](https://usegalaxy.org.au/).
```

Comment:
```suggestion
 There are 3 main public Galaxy servers: [UseGalaxy.org](https://usegalaxy.org/), [UseGalaxy.eu](https://usegalaxy.eu/), and [UseGalaxy.org.au](https://usegalaxy.org.au/).
```

Reply:
@bernandez please look here, there are suggestions not applied, you can click on "Commit suggestion" and get it done...